### PR TITLE
Problem: tally calculation fix is not used

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- upgrade cosmos-sdk for the fix

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-com/chain-main/blob/master/CONTRIBUTING.md)?
- [x] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [x] Have you rebased your work on top of the latest master? 
- [x] Have you checked your code compiles? (`make`)
- [x] Have you included tests for any non-trivial functionality?
- [x] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-com/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-com/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-com/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Comment by yihuang** [comment]
https://github.com/crypto-com/chain-main/issues/239
the test failure record here.

**Comment by tomtau** [comment]
> #239
> the test failure record here.

is it expected -- due to the fix point calculation changes?

**Comment by yihuang** [comment]
> > #239
> > the test failure record here.
> 
> is it expected -- due to the fix point calculation changes?

the current fix is only changing order or operation in tally calculation, the bigger fix point calculation refactor is still in discuss, should have no effect on record, I'm re-running the tests to see if it works.

**Comment by yihuang** [comment]
re-running the test is fine

File: cmd/chain-maind/app/app.go [hunk 1]
```diff
@@ -201,6 +202,10 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 	)
 }
 
+func addModuleInitFlags(startCmd *cobra.Command) {
+	crisis.AddModuleInitFlags(startCmd)
```
Comment (tomtau) [inline cmd/chain-maind/app/app.go:206]: do we use the crisis module?

Comment (yihuang) [reply cmd/chain-maind/app/app.go:206]: no, i'll remove it.

Comment (yihuang) [reply cmd/chain-maind/app/app.go]: done

**Review by tomtau — APPROVED** [review]

**Pull request merged**